roles/ee_builder/tasks/00_build_ee.yml

```diff
@@ -0,0 +1,90 @@
+---
+# tasks file for ee_builder
+- name: Build directory creation block.
+  block:
```

Comment:
Could this not be done in `main.yml` just to save needing to go through this each time? (I know it won't actually run each time but it will still need to evaluate whether `builder_dir` is defined and will show up skipped.